templates/matterbridge.yaml.j2

```
@@ -65,8 +65,9 @@ gateway:
     {% endfor %}
     {% for discord_srv in gtw.get("discord", []) %}
 
+      # for channel name : "{{ discord_srv.ch }}"
       - account: "discord.{{ discord_srv.srv }}"
-        channel: "{{ discord_srv.ch }}"
+        channel: "ID:{{ discord_srv.id }}"
```

Why not support both?
```suggestion
        channel: "{{ 'ID:'+discord_srv.id if discord_srv.id else discord_srv.ch }}"
```

---

I was reading somewhere that its got to be either all channel IDs or all channel names and a mix match might break the bridge, probably due to the autowebhooks setting.

---

I see, I wasn't aware of this. If that's the case then we should indeed only support ID.

---

my bad the issue I described was fixed : https://github.com/42wim/matterbridge/commit/8d9bec366eee54090bc2f574d13bf3f0d7d02f97 so we can support both